### PR TITLE
feat: add scroll for table

### LINKED ISSUE #423: 移动版页面，宽表格显示不全，且无法滑动
## Describe the bug
移动浏览器访问页面中的表格，表格显示不全而且无法像代码一样滑动

## To Reproduce
Indicate the steps to reproduce this bug, if applicable.

## Screenshots
Add screenshots to help explain your problem.

## Environment

- **Hugo version** [e.g: 0.80.0]: 最新版的
- **Hugo extended?**: No 

## Additional context

If it's a UI issue, fill the following information:

 - OS: [e.g. iOS] 安卓
 - Browser [e.g. chrome, safari] Chrome 移动端
 - Version [e.g. 22] 最新版

## Content of config.yaml
```yaml
### Paste the content of the config file here
没修改过跟元素相关的内容
```

### PR DESCRIPTION
**Pull request by zhixuan2333**
fix #423.

Change:
- Removed scroll style for Chromium. Change to color-scheme.
- Added div element for table

Example for actions:

https://user-images.githubusercontent.com/59254886/146128576-506e4936-4b03-4fc7-a397-4c50b0fdb8d4.mp4

### CODE REVIEW

File: assets/scss/partials/base.scss [hunk 1]
```diff
@@ -23,17 +23,3 @@ body {
...
-}
-
-::-webkit-scrollbar-thumb {
-    background-color: var(--scrollbar-thumb);
-}
-
-::-webkit-scrollbar-track {
-    background-color: transparent;
-}
-/**/
```
Comment (CaiJimmy) [inline assets/scss/partials/base.scss:39]: Why is this style removed?

**Comment by CaiJimmy** [comment]
I'll merge this. But probably I'll create another PR in the future, replacing this Regex replace with a JavaScript based solution (wrap `<table>` with div.table-wrapper).

File: assets/scss/partials/base.scss (hunk 1 above)
Comment (zhixuan2333) [reply assets/scss/partials/base.scss:39]: image
image

删除之后
image

首先是因为根本看不见...  然后就是theme的scroll bar不怎么好看

Comment (CaiJimmy) [reply assets/scss/partials/base.scss:39]: 不用，就这样吧 👍

**Review by CaiJimmy — APPROVED** [review]

**Pull request merged**